nixos/modules/programs/ydotool.nix

```
@@ -14,23 +14,32 @@ in
 
   options.programs.ydotool = {
     enable = lib.mkEnableOption ''
-      ydotoold system service and install ydotool.
-      Add yourself to the 'ydotool' group to be able to use it.
+      Enable ydotoold system service and {command}`ydotool` for members of
```

https://github.com/NixOS/nixpkgs/blob/78be25a21623159fac10384f2c245e7161b49fe4/lib/options.nix#L98-L103

```suggestion
      ydotoold system service and {command}`ydotool` for members of
```